qiskit_experiments/framework/base_experiment.py

```diff
@@ -184,7 +207,10 @@ def run(
             backend: Optional, the backend to run the experiment on. This
                      will override any currently set backends for the single
                      execution.
-            analysis: If True run analysis on the experiment data.
+            analysis: Optional, a custom analysis instance to use for performing
+                      analysis. If None analysis will not be run. If ``"default"``
+                      the experiments :meth:`analysis` instance will be used if
+                      it contains one.
```

Comment:
```suggestion
            analysis: Optional, a custom analysis instance to use for performing
                      analysis. If ``None`` or ``False`` analysis will not be run. 
                      If ``"default"`` or ``True`` the experiments :meth:`analysis` 
                      instance will be used if it contains one.
```

Reply:
I would rather move away from using `False` and use `None`, since `exp.analysis = None` is valid syntax to remove analysis instance.

Reply:
But relasenote says booleans are still valid. I'm fine with updating the node and remove boolean usage